doc/rest_api/snippet_sharing/use_cases/UC_edit_user.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: edit user
+Scope: User Management
+Level: User Goal
+Actors: Authenticated user, Administrator
+Brief: Users can edit their own account information (e.g. change password). Administrators can change all user accounts in the database. Administrators can also change permissions of users.

```

Comment:
You are right, a password reset function is missing, because currently I've no idea how to deal with emails yet. Could also be an idea to add email verification during registration (that doesn't really change the API anyway).

The problem with spammers is that they can simply create a new account and spam on. It is basically the same problem you have on any platform. Absolutely no idea how to prevent that without taking away write permissions by default.